trajoptlib/src/lib.rs

```diff
@@ -683,7 +687,11 @@ impl DifferentialPathBuilder {
     ) -> Result<DifferentialTrajectory, TrajoptError> {
         match self.path_builder.generate(diagnostics, handle) {
             Ok(traj) => Ok(traj),
-            Err(msg) => Err(TrajoptError::from(msg.what().parse::<i8>().unwrap())),
+            Err(msg) => {
+                let what = msg.what();
+                println!("Trajoptlib Error: {}", what);
+                Err(TrajoptError::from(what.parse::<i8>().unwrap_or(-128)))
```

Comment:
Does the parse failure mainly happen if the wrong type got thrown by C++?

Reply:
Thats exclusively when it happens, i prolly coulda done a
`map_err(|_| TrajoptError::Unparsable)?` instead of `unwrap_or(-128)`